src/components/InventoryModal/InventoryModal.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { Button, Modal, ModalContent } from 'semantic-ui-react'
+import AddItem from '../AddItem'
+import SoldItem from '../SoldItem'
+import ModifyItem from '../ModifyItem'
+import RemoveItem from '../RemoveItem'
+import '../../styles/modal.css'
+
+const modalTypes = {
```

Comment:
maybe we can extract this to a constants file?

Reply:
In fact, we probably don't even need this configuration file. See the another comments below